### PR TITLE
fixe(conf) : change scenes sent config and on/off mode - fibaro walli dimmer

### LINKED ISSUE #2424: Scenes sent readOnly - Fibaro Walli Dimmer #config
Hi,

There are errors in the configuration of Fibaro Walli Dimmer :
First button – scenes sent is readOnly
Second button – scenes sent is readOnly
Change On/Off mode has the id 0

### PR DESCRIPTION
**Pull request by Micka41**
fixes: #2424

### CODE REVIEW

**Comment by Micka41** [comment]
@zwave-js-bot fix lint

**Comment by Micka41** [comment]
@zwave-js-bot fix lint

**Comment by Micka41** [comment]
@zwave-js-bot fix lint

**Comment by Micka41** [comment]
@zwave-js-bot fix lint

**Comment by Micka41** [comment]
@zwave-js-bot fix 2424

File: packages/config/config/devices/0x010f/fgwdeu.json [hunk 1]
```diff
@@ -310,9 +300,31 @@
...
+				},
+				{
+					"label": "Key pressed 3 times",
+					"value": 4
+				},
+				{
+					"label": "Key held down and released",
+					"value": 8
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x010f/fgwdeu.json:327]: These options should be modeled as partial parameters so they can be combined with each other. The other parameters that have the same options aren't correct either.

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Thanks! Unfortunately this device file seems to come from an earlier automated import from other projects and contains some more errors and/or needs cleaning up.

Since you touched it, we should make sure it is up to our standards. Please read the [Style Guide](https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide) and make the necessary corrections where necessary.

If you have any questions, @blhoward2 and I will gladly assist you.

**Pull request closed without merging**